### ISSUE TITLE
UncheckedIOException: java.nio.file.NoSuchFileException

### ISSUE THREAD

**Issue by pihme** [camunda-cloud/zeebe#5488]
**Describe the bug**
Observed in production log.

**Log/Stacktrace**
https://console.cloud.google.com/errors/CM3_pPb5yeCowwE?service=zeebe&time=P7D&project=camunda-cloud-240911

**Environment:**
Camunda Cloud / Prod / 0.24.3

**Comment by npepinpe** [comment]
Let's investigate this as soon as possible - expected impact is that you cannot delete data, but it's unclear if this happens often or not to the same clusters. So let's answer the following:

1. How often in a given cluster does it happen?
1. Is it transient (does it resolve upon restart)?
1. Confirm that the impact is what I stated (and not worse)

**Comment by npepinpe** [comment]
I had a look, this happened because we tried to take a snapshot, which failed due to out of disk space - we then tried to roll this back, which failed because we couldn't delete the snapshot that we never took.

So impact is minimal, other than noise in terms of errors. What I would like here is that we handle this case - if we failed to create the snapshot, there should be nothing to roll back (or at least, don't try to delete it).
